dev/com.ibm.ws.security.javaeesec_fat/test-applications/dbfatAuthAlias.war/src/web/war/db/CustomQueryDatabaseServlet.java

```diff
@@ -24,40 +24,55 @@
         try {
             DataSource ds = (DataSource) new InitialContext().lookup("jdbc/derby1fat");
 
+            Connection conn = ds.getConnection();
+            try {
+                Statement stmt1 = conn.createStatement();
+                stmt1.executeUpdate("create schema dbuser1");
+                stmt1.close();
+            } finally {
+                conn.close();
+            }
+
             FATDatabaseServlet.createTable(ds, callerTable, "name varchar(30), password varchar(300)");
 
             FATDatabaseServlet.createTable(ds, callerGroups, "group_name VARCHAR(36), caller_name VARCHAR(36)");
 
-            Connection conn = ds.getConnection();
-            Statement stmt1 = conn.createStatement();
-            stmt1.executeUpdate("insert into " + callerTable + " (password, name) values ('" + Constants.DB_USER1_PWD_HASH + "' , '" + Constants.DB_USER1 + "')");
-            stmt1.close();
+            conn = ds.getConnection();
+            try {
+                Statement stmt1 = conn.createStatement();
+                stmt1.executeUpdate("insert into " + callerTable + " (password, name) values ('" + Constants.DB_USER1_PWD_HASH + "' , '" + Constants.DB_USER1 + "')");
+                stmt1.close();
 
-            stmt1 = conn.createStatement();
-            stmt1.executeUpdate("insert into " + callerGroups + " (group_name, caller_name) values ('" + Constants.DB_GROUP2 + "' , '" + Constants.DB_USER1 + "')");
-            stmt1.close();
+                stmt1 = conn.createStatement();
+                stmt1.executeUpdate("insert into " + callerGroups + " (group_name, caller_name) values ('" + Constants.DB_GROUP2 + "' , '" + Constants.DB_USER1 + "')");
+                stmt1.close();
 
-            stmt1 = conn.createStatement();
-            stmt1.executeUpdate("insert into " + callerTable + " (password, name) values ('" + Constants.DB_USER2_PWD_HASH + "' , '" + Constants.DB_USER2 + "')");
-            stmt1.close();
+                stmt1 = conn.createStatement();
+                stmt1.executeUpdate("insert into " + callerTable + " (password, name) values ('" + Constants.DB_USER2_PWD_HASH + "' , '" + Constants.DB_USER2 + "')");
+                stmt1.close();
 
-            stmt1 = conn.createStatement();
-            stmt1.executeUpdate("insert into " + callerGroups + " (group_name, caller_name) values ('" + Constants.DB_GROUP3 + "' , '" + Constants.DB_USER2 + "')");
-            stmt1.close();
+                stmt1 = conn.createStatement();
+                stmt1.executeUpdate("insert into " + callerGroups + " (group_name, caller_name) values ('" + Constants.DB_GROUP3 + "' , '" + Constants.DB_USER2 + "')");
+                stmt1.close();
 
+            } finally {
+                conn.close();
+            }
         } catch (Exception e) {
             System.out.println("Failed to create database for DatabaseIdentityStore: " + e.getMessage());
             e.printStackTrace();
+        } finally {
+
```

Comment:
Did we intend to have an empty finally block?

Reply:
Removed empty finally block.